relay/paseo/src/genesis_config_presets.rs

```diff
@@ -0,0 +1,249 @@
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Polkadot.  If not, see <http://www.gnu.org/licenses/>.
+
+//! Genesis configs presets for the Polkadot runtime
+
+use crate::*;
+use babe_primitives::AuthorityId as BabeId;
+use pallet_staking::{Forcing, StakerStatus};
+use polkadot_primitives::{AccountPublic, AssignmentId, AsyncBackingParams};
+use paseo_runtime_constants::currency::UNITS as PAS;
+use runtime_parachains::configuration::HostConfiguration;
+use sp_core::{sr25519, Pair, Public};
+use sp_runtime::{traits::IdentifyAccount, Perbill};
+#[cfg(not(feature = "std"))]
+use sp_std::alloc::format;
+use sp_std::vec::Vec;
+
+/// Helper function to generate a crypto pair from seed
+fn get_from_seed<TPublic: Public>(seed: &str) -> <TPublic::Pair as Pair>::Public {
+	TPublic::Pair::from_string(&format!("//{}", seed), None)
+		.expect("static values are valid; qed")
+		.public()
+}
+
+/// Helper function to generate an account ID from seed
+fn get_account_id_from_seed<TPublic: Public>(seed: &str) -> AccountId
+where
+	AccountPublic: From<<TPublic::Pair as Pair>::Public>,
+{
+	AccountPublic::from(get_from_seed::<TPublic>(seed)).into_account()
+}
+
+/// Helper function to generate stash, controller and session key from seed
+fn get_authority_keys_from_seed(
+	seed: &str,
+) -> (
+	AccountId,
+	AccountId,
+	BabeId,
+	GrandpaId,
+	ValidatorId,
+	AssignmentId,
+	AuthorityDiscoveryId,
+	BeefyId,
+) {
+	(
+		get_account_id_from_seed::<sr25519::Public>(&format!("{}//stash", seed)),
+		get_account_id_from_seed::<sr25519::Public>(seed),
+		get_from_seed::<BabeId>(seed),
+		get_from_seed::<GrandpaId>(seed),
+		get_from_seed::<ValidatorId>(seed),
+		get_from_seed::<AssignmentId>(seed),
+		get_from_seed::<AuthorityDiscoveryId>(seed),
+		get_from_seed::<BeefyId>(seed),
+	)
+}
+
+fn testnet_accounts() -> Vec<AccountId> {
+	vec![
+		get_account_id_from_seed::<sr25519::Public>("Alice"),
+		get_account_id_from_seed::<sr25519::Public>("Bob"),
+		get_account_id_from_seed::<sr25519::Public>("Charlie"),
+		get_account_id_from_seed::<sr25519::Public>("Dave"),
+		get_account_id_from_seed::<sr25519::Public>("Eve"),
+		get_account_id_from_seed::<sr25519::Public>("Ferdie"),
+		get_account_id_from_seed::<sr25519::Public>("Alice//stash"),
+		get_account_id_from_seed::<sr25519::Public>("Bob//stash"),
+		get_account_id_from_seed::<sr25519::Public>("Charlie//stash"),
+		get_account_id_from_seed::<sr25519::Public>("Dave//stash"),
+		get_account_id_from_seed::<sr25519::Public>("Eve//stash"),
+		get_account_id_from_seed::<sr25519::Public>("Ferdie//stash"),
+	]
+}
+
+fn default_parachains_host_configuration() -> HostConfiguration<polkadot_primitives::BlockNumber> {
+	use polkadot_primitives::{MAX_CODE_SIZE, MAX_POV_SIZE};
+
+	runtime_parachains::configuration::HostConfiguration {
```

Comment:
This appears to be the same as the one I reviewed in chat, worth checking the three that I highlighted:
- nodeFeatures are not set, which I think is required for some async backing functionality
- on-demand queue looks a fair bit bigger than polkadot and may lead to worse performance/higher weights. you might want to just double check that before using it
- ump limits are quite low, they should still be fine for coretime messages with the number of parachains you have, but some day that might be an issue